homeassistant/components/imeon_inverter/sensor.py

```diff
@@ -0,0 +1,79 @@
+"""Imeon inverter sensor support."""
+
+import logging
+
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import InverterCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Create each sensor for a given config entry."""
+
+    # Get Inverter from UUID
+    IC: InverterCoordinator = entry.runtime_data
```

Comment:
```suggestion
    coordinator = entry.runtime_data
```
is then inferred typed